include/effects/matrix/PatternWeather.h

```diff
@@ -104,30 +110,70 @@ class PatternWeather : public LEDStripEffect
         return (K - 273.15) * 9.0f/5.0f + 32;
     }
 
-    inline float KelvinToCelcius(float K)
+    inline float KelvinToCelsius(float K)

```

Comment:
You'd think, being Canadian, I could have spelled it more goodly.

Reply:
:) I missed it too in my review of the initial conversion code.